rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -288,29 +288,31 @@
               class="dateStarted date"
               colspan="2"
             >
+              <!--getting vue warning: Invalid prop: type check failed for prop "modelValue".Converted :value to :modelValue-->
               <progress-bar
                 v-if="exec.status === 'scheduled'"
-                :value="100"
+                :modelValue="100"
                 striped
                 type="default"
                 label
                 :label-text="
                   $t('job.execution.starting.0', [
-                    runningStartedDisplay(exec.dateStarted.date),
+                    // if exec.dateStarted is null or undefined,it doesn't throw an error but instead returns undefined
+                    runningStartedDisplay(exec.dateStarted?.date) || 'N/A',
```

Comment:
Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

Reply:
> Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

I was going through line by line code between progress bar (first I saw in vue dev tool to confirm which line of code I have to see to make it  right) and started understanding the functions. So the `runningStartedDispaly` requires the valid date to return the right output.

Reply:
> Question, what led you to the conclusion that this change was needed? Have you seen a scenario where this happens?

Ok so I fixed it now. It was just unnecessary change from my side. Ignore it